PSMenu/Public/Show-Menu.ps1

```diff
@@ -140,6 +147,12 @@ function Show-Menu {
                 [System.Console]::SetCursorPosition(0, [Console]::CursorTop - $MenuHeight)
                 & $WriteMenu
             }
+
+            if ($Callback) {
+                & $Callback
+            }
+            
+            Start-Sleep -Milliseconds 100
```

Comment:
The 100 millisecond delay may make navigation of large menu's even more cumbersome than it is. Decrease to 10 milliseconds? Then any debouncing can happen in there.

Perhaps the callback is best called from `Read-VKey` or a wrapper around that function, to because this entire `while` loop tends to re-render the menu.

Reply:
I reduced it to 10 milliseconds.
Re-rendering the menu is intentional, I use this in one place to display a live status beside each menu item